west.yml

```diff
@@ -23,6 +23,8 @@ manifest:
       url-base: https://github.com/zephyrproject-rtos
     - name: babblesim
       url-base: https://github.com/BabbleSim
+    - name: kholia
+      url-base: https://github.com/kholia
```

Comment:
Could you send https://github.com/kholia/hal_wch/commit/787d57f0effc84205d2ea10489b7fe7cd1c4cded up to https://github.com/cnlohr/ch32v003fun and then use https://github.com/cnlohr/ch32v003fun directly here? Sounds like @cnlohr is happy to have that file there. The rest looks good, I'll rework the proposal and bring it up to the TSC.

Reply:
Sure. https://github.com/cnlohr/ch32v003fun/pull/364 is open now for @cnlohr's review.

Reply:
Great stuff, you can already test it on the PR by setting this to `https://github.com/cnlohr` and the one below to `ch32v003fun` and revision to `pull/364/head`.

Reply:
Done. I will re-push once the PR (https://github.com/cnlohr/ch32v003fun/pull/364) is merged.

I have also added @nzmichaelh @kholia (myself) to `MAINTAINERS.yml` file now.

Reply:
Oh, I'm sorry, I just saw this discussion here after commenting.  That makes more sense - what is the normal process for updating to newer versions of sub repos?  Is the idea just keep it pinned until there are bugs?

Side-note: PR 346 has been accepted to 003fun.